src/Diagnostics.Scripts/EntityInvoker.cs

```diff
@@ -125,9 +127,8 @@ public void InitializeEntryPoint(Assembly asm)
             }
             catch (Exception ex)
             {
-                if (ex is ScriptCompilationException)
+                if (ex is ScriptCompilationException scriptEx)
```

Comment:
Why not just do
```
catch(ScriptCompilationException scriptEx) {
   ...
}
```

Wouldn't that accomplish the same thing?

Reply:
ya, I was planning to add more custom exception types like invocationexception to pass through the invocation errors a part of the api